scripts/makefile/patchval.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+# Files addition to watch for
+PATCH_COUNT=$(git show --name-status | grep "^A" | grep -c "\.patch" )
```

Comment:
git show displays only last commit, for CI better to pass and previous commit to compare to

Reply:
Good catch, gonna dig that

Reply:
Fixed!